estate/models/estate_property.py

```diff
@@ -0,0 +1,33 @@
+from datetime import datetime, timedelta
+from odoo import fields, models
+
+class estateModel(models.Model):
+    _name = "estate.property"
+    _description = "Estate testing model"
+
+    id = fields.Integer()
+    name = fields.Char('Estate Name', required=True)
```

Comment:
The string for the fields should be appropriate. It should provide be informative so that it would be more user friendly
```suggestion
    name = fields.Char('Name', required=True)
```

Reply:
Okay sir, changes done.